### PR TITLE
[daml-triggers] Support wallclock time and make TTL configurable

### LINKED ISSUE #3184: Make command times in DAML triggers configurable
Currently, DAML triggers only work with EPOCH time and a fixed TTL. We should add a `-w` flag to run in wallclock mode and a parameter to configure the TTL.

### PR DESCRIPTION
**Pull request by cocreature**
fixes #3184

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/unreleased.rst), if appropriate
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: triggers/runner/src/main/scala/com/daml/trigger/RunnerConfig.scala [hunk 1]
```diff
@@ -40,6 +46,24 @@ object RunnerConfig {
...
+      .action { (t, c) =>
+        c.copy(timeProviderType = TimeProviderType.WallClock)
+      }
+      .text("Use wall clock time (UTC). When not provided, static time is used.")
+
+    opt[Unit]('w', "wall-clock-time")
+      .action { (t, c) =>
+        c.copy(timeProviderType = TimeProviderType.WallClock)
+      }
+      .text("Use wall clock time (UTC). When not provided, static time is used.")
```
Comment (aherrmann-da) [inline triggers/runner/src/main/scala/com/daml/trigger/RunnerConfig.scala]: Looks like a duplicate. Am I missing something?

File: triggers/runner/src/main/scala/com/daml/trigger/RunnerConfig.scala [hunk 2]
```diff
@@ -49,5 +73,9 @@ object RunnerConfig {
         triggerIdentifier = null,
         ledgerHost = "",
         ledgerPort = 0,
-        ledgerParty = ""))
+        ledgerParty = "",
+        timeProviderType = TimeProviderType.Static,
+        commandTtl = Duration.ofSeconds(30L),
```
Comment (aherrmann-da) [inline triggers/runner/src/main/scala/com/daml/trigger/RunnerConfig.scala]: What prompts the change from 5s to 30s? 
(Just curious)

File: triggers/runner/src/main/scala/com/daml/trigger/RunnerConfig.scala (hunk 1 above)
Comment (cocreature) [reply triggers/runner/src/main/scala/com/daml/trigger/RunnerConfig.scala]: Good catch, thanks! Fixed

File: triggers/runner/src/main/scala/com/daml/trigger/RunnerConfig.scala (hunk 2 above)
Comment (aherrmann-da) [reply triggers/runner/src/main/scala/com/daml/trigger/RunnerConfig.scala]: :+1: Thanks for explaining!

**Pull request merged**